### DISCUSSION TITLE
Chrome os do not recognize macbook air webcam

### DISCUSSION THREAD

**Discussion by G-Ventur4** [sebanc/brunch#1007] [Q&A]
Hi, i have installed the latest brunch frame work and the latest chrome os build, and i can´t use the webcam, can anyone help me ? I have a macbook air 2015

**Comment by LillyYaGirl** [comment]
I had the same problem with my 2012 macbook pro. open the shell (control + alt + T), type "shell" and hit enter, then type "sudo edit-grub-config" and it will let you edit the grub config file, where you want to type "no_camera_config" right after "options=" (NO SPACES).
